multiaccounts/settings/columns.go

```diff
@@ -185,9 +189,22 @@ var (
 		},
 	}
 	Mnemonic = SettingField{
-		reactFieldName: "mnemonic",
-		dBColumnName:   "mnemonic",
+		reactFieldName: DBColumnMnemonic,
+		dBColumnName:   DBColumnMnemonic,
 	}
+
+	MnemonicBackedUp = SettingField{
+		reactFieldName: "mnemonic-backed-up?",
+		dBColumnName:   "mnemonic_backed_up",
```

Comment:
The term `MnemonicBackedUp` seems to communicate that mnemonic data *can* be backed up. But what we really are checking is if `mnemonic` is empty. This language could cause confusion with other developers who may believe that the mnemonic value should be backed up.

Perhaps change this field name to something like `MnemonicRemoved`

Reply:
fixed